### ISSUE TITLE
Add mutation examples

### ISSUE THREAD

**Issue by ivan-kolmychek** [joshprice/hello_graphql_phoenix#9]
It would be nice to have some examples of changing data through graphql mutations.

**Comment by joshprice** [comment]
Yes absolutely. This is definitely missing and we'll add these in this week.

**Comment by ivan-kolmychek** [comment]
@joshprice thank you. While I got the general idea from looking at the code itself and examples for other languages, I think it would benefit anyone who explores the GraphQL through this example repo to be able to see the examples for mutations, clone them to their machine and play around to see how do they work.

**Comment by joshprice** [comment]
It's definitely our top priority, thanks for the bug report. 🙏

**Comment by mechabyte** [comment]
Is there an update on the progress of these code examples?

**Comment by 0x6e6562** [comment]
Has anybody come across any mutation examples in external repos?

**Comment by ivan-kolmychek** [comment]
@0x6e6562 there are some examples in repos from official [GraphQL Github organization](https://github.com/graphql/).
- The `graphql-js` [has multiple mentions of them](https://github.com/graphql/graphql-js/search?utf8=%E2%9C%93&q=mutations) 
- The `express-graphql` has one [example definition of empty mutation](https://github.com/graphql/express-graphql/blob/0bf6aac9ec1c02d42f5de93ff068182c304d56a4/src/__tests__/http-test.js#L58)

**Comment by ivan-kolmychek** [comment]
The `graphql-relay-js` repo [has multiple mentions](https://github.com/graphql/graphql-relay-js/search?utf8=%E2%9C%93&q=mutation) too.

**Comment by ivan-kolmychek** [comment]
Also, here is the post on Medium about mutations: https://medium.com/@tarkus/validation-and-user-errors-in-graphql-mutations-39ca79cd00bf

**Comment by ivan-kolmychek** [comment]
[The docs from `graphql-relay-ruby` also mention mutations](https://github.com/rmosolgo/graphql-relay-ruby#mutations)

Quote (empathis mine):

> Mutations allow Relay to mutate your system. They conform to a strict API which makes them predictable to the client.
> ...
> Members of MutationType are mutation fields. **For GraphQL in general, mutation fields are identical to query fields except that they have side-effects (which mutate application state, eg, update the database).**
> For Relay-compliant GraphQL, a mutation field must comply to a strict API. GraphQL::Relay includes a mutation definition helper (see below) to make it simple.

So, there are 1) regular GraphQL mutations, which are queries that have side-effects like modifying the database and 2) Relay mutations that are somehow stricter in terms of API.

**Comment by kespinola** [comment]
I'm also wondering how to handle file uploads. Sending file using relay `getFiles()`, but I don't have access to the param holding the file upload. 

Seeing no info here in regards to `%Plug.Upload` https://github.com/graphql-elixir/graphql/blob/master/lib/graphql/execution/executor.ex#L141
